tools/local_provisioning_profile_finder/local_provisioning_profile_finder.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import datetime
+import plistlib
+import shutil
+import subprocess
+from typing import List, Optional, Tuple
+
+
+def _build_parser() -> argparse.ArgumentParser:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("name", help="The name of the profile to find")
+    parser.add_argument("output", help="The path to copy the profile to")
+    parser.add_argument(
+        "--local-profiles",
+        nargs="*",
+        required=True,
+        help="All local provisioning profiles to search through",
+    )
+    parser.add_argument(
+        "--fallback-profiles",
+        nargs="+",
+        required=True,
+        help="Fallback provisioning profiles to use if not found locally",
+    )
+    return parser
+
+
+def _profile_contents(profile: str) -> Tuple[str, datetime.datetime]:
+    output = subprocess.check_output(["security", "cms", "-D", "-i", profile])
```

Comment:
I met an issue running `security` on CI and was able to prevent the permission error with the following change : https://github.com/autopkg/recipes/commit/657b2eb51f22dd14d44b90a0ae7a1d3e3313087e

Reply:
Ouch what was the error you saw? we successfully run this on CI today, with the standard keychain setup workarounds required for signing. So if I understand correctly your workaround is to create a dummy keychain and point the command to that?

Reply:
The error we saw was : 
```
security: cert import failed: write permissions error
security: problem decoding
```

Based on the context I read about why this issue happens, it's possible that the error could happen due to missing field in the `.mobileprovision` file being passed to the `security` command. So, it's also possible that you don't need this dummy keychain in Lyft's environment. 

> So if I understand correctly your workaround is to create a dummy keychain and point the command to that?
Yes

Reply:
Don't you also have a keychain setup on CI to do the actual signing? We're just relying on that one I guess